### PR TITLE
Un-commented all lines about intake pivot motor & added intake absolute encoder

### PR DESCRIPTION
**Pull request by Hyrumthehotscott**
Issue #114

### CODE REVIEW

**Comment by TaylerUva** [comment]
Wait to merge this in until bot is build but looks good

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 1]
```diff
@@ -7,27 +7,39 @@
...
+import frc.robot.Constants.constIntake;
 import frc.robot.RobotMap.mapIntake;
 import frc.robot.RobotPreferences.prefIntake;
 
 public class Intake extends SubsystemBase {
   TalonFX rollerMotor;
-  // TalonFX pivotMotor;
+  TalonFX pivotMotor;
+  CANSparkMax leftCenteringMotor;
+  CANSparkMax rightCenteringMotor;
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Intake.java]: These don't exist anymore

File: src/main/java/frc/robot/RobotMap.java [hunk 2]
```diff
@@ -52,6 +54,9 @@ public static class mapDrivetrain {
   public static class mapIntake {
     public static final int INTAKE_ROLLER_MOTOR_CAN = 20;
     public static final int INTAKE_PIVOT_MOTOR_CAN = 23;
+    public static final int INTAKE_ABSOLUTE_ENCODER = 24;
+    public static final int INTAKE_LEFT_CENTERING_MOTOR_CAN = 25;
+    public static final int INTAKE_RIGHT_CENTERING_MOTOR_CAN = 26;
```
Comment (TaylerUva) [inline src/main/java/frc/robot/RobotMap.java]: these don't exist anymore

**Review by TaylerUva — CHANGES_REQUESTED** [review]

**Comment by Hyrumthehotscott** [comment]
😢

**Pull request closed without merging**